src/login/logind-session.h

```diff
@@ -89,6 +89,7 @@ struct Session {
         int vtfd;
 
         PidRef leader;
+        bool leader_fd_saved;
```

Comment:
can you add brief comment here what this means, i.e. just suffix `/* pidfd of leader uploaded to fdstore */` or so?